clinicaltrials/frontend/management/commands/load_data.py

```diff
@@ -526,20 +531,18 @@ def convert_to_csv():
             td["pending_results"] = does_it_exist(soup.pending_results)
 
             td["pending_data"] = dict_or_none(parsed_json, [cs, "pending_results"])
+                                  
+            date_cond1 = td["available_completion_date"] + relativedelta(years=1) + timedelta(days=30)
+            
+            date_cond2 = td["available_completion_date"] + relativedelta(years=3) + timedelta(days=30)                    
```

Comment:
Did you just make this change for readability?

Reply:
Sort of. Those conditions got moved around a bit from elsewhere in the code and I was worried the indents were not correct so I just moved them all to 1 line.

Seems that wasn't the case though.